lib/taurus/core/evaluation/evalattribute.py

```diff
@@ -484,7 +484,7 @@ def addListener(self, listener):
         if len(self._listeners) > 1 and \
            (initial_subscription_state == SubscriptionState.Subscribed or
                 self.isPollingActive()):
-            Manager().enqueueJob(self.__fireRegisterEvent, None, (listener,))
+            Manager().enqueueJob(self.__fireRegisterEvent, None, ((listener,)))
```

Comment:
It would be better to use kwargs explicitly. 
Also, the extra parentheses added are superfluous (note that since there is no `,`, they do not denote a tuple but just a superfluous priority).
In summary, it should be:
`Manager().enqueueJob(self.__fireRegisterEvent, callback=None, jobargs=(listener,))`